inc/spbc-settings.php

```diff
@@ -2324,6 +2414,51 @@ function spbc_field_traffic_control_logs__prepare_data(&$table)
     }
 }
 
+/**
+ * To draw a banner inside spbct tabs
+ * @param string $flag_text_banner which banner text to show
+ */
+function spbct_admin_notice($flag_text_banner)
```

Comment:
please name this function correct - name should contain the action that function does, something like spbc_settings__show_notice_on_tab()

Reply:
2. Provide @return void in docblock and describe what the function does